predicators/envs/behavior.py

```diff
@@ -756,6 +761,8 @@ def _closed_classifier(self, state: State, objs: Sequence[Object]) -> bool:
     @staticmethod
     def _ig_object_name(ig_obj: "ArticulatedObject") -> str:
         if isinstance(ig_obj, (URDFObject, RoomFloor)):
+            if "board_game" in ig_obj.name:
+                return ig_obj.name + ".n.01_1"
```

Comment:
i'm so confused; how are board_game objects of type RoomFloor?

Reply:
That is an isinstance of URDFObject or RoomFloor.